releases/release-1.26/release-notes/maps/pr-112748-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 112748
+releasenote:
+  text: Lock `ServerSideApply` feature gate to true with the feature already being
```

Comment:
Locked `ServerSideApply` feature gate to true with the feature already being

Reply:
/Ack